devel/jam/Portfile

```diff
@@ -62,7 +59,7 @@ use_configure       no
 
 pre-build {
     build.cmd-append    CC="${configure.cc}" \
-                        CFLAGS=\"[get_canonical_archflags]\"
+                        CFLAGS=\"[get_canonical_archflags] -Wno-implicit-function-declaration\"
```

Comment:
it would be preferable to fix the implicit declaration instead of turning off the warnings...

Reply:
Yes, agreed, that sounds like a better solution. Not really within my ability to patch though. (I just wanted the port working so it could build another port I needed!)

I did poke around a bit and found the header that was causing problems (<unistd.h> I think), but I couldn't figure out why. Let me take another look, see if I can do better. I'll report back either way.

Reply:
OK, that wasn't so hard! Thanks for the push.